mock/client/getMessage.js

```diff
@@ -0,0 +1,12 @@
+'use strict'
```

Comment:
in general it's good to have block comments at the beginning of each source file that explain the purpose of that file